hummingbird/ml/operator_converters/skl_decision_tree.py

```diff
@@ -47,31 +40,10 @@ def convert_sklearn_random_forest_classifier(operator, device, extra_config):
     if not all(isinstance(c, int) for c in classes):
         raise RuntimeError("Random Forest Classifier translation only supports integer class labels")
 
-    tree_parameters, max_depth, tree_type = get_tree_params_and_type(
-        tree_infos, get_parameters_for_sklearn_common, extra_config
+    return convert_decision_ensemble_tree_common(
+        tree_infos, get_parameters_for_sklearn_common, get_parameters_for_tree_trav_sklearn, n_features, classes, extra_config
     )
 
-    # Generate the tree implementation based on the selected strategy.
```

Comment:
were these moved to make sense with the onnxml trees as well? I am a bit lost here and need more info.

Reply:
We already merged a PR with this changes actually (#132).